src/pynorare/cli_util.py

```diff
@@ -1,17 +1,21 @@
-from pyconcepticon import Concepticon
-
-from pynorare.dataset import get_dataset_cls
+import pathlib
 
 
 def add_datasets(parser):
     parser.add_argument(
         'dataset',
-        nargs='+',
+        nargs='*',
         help='select your dataset',
         type=str)
+    parser.add_argument('--start-from', default=None)
 
 
 def iter_datasets(args):
-    for dsid in args.dataset:
-        cls = get_dataset_cls(args.api.datasets[dsid].path.parent)
-        yield cls(repos=args.norarepo, concepticon=Concepticon(args.repos.repos))
+    dsids = [
+        pathlib.Path(dsid).name if pathlib.Path(dsid).exists() else dsid for dsid in args.dataset]
+    for dsid, ds in sorted(args.api.datasets.items(), key=lambda i: i[0]):
```

Comment:
For the `--start-from` option to work, we always have to loop through datasets in alphabetic order of ID.